src/i18n/Localizable.xcstrings

```diff
@@ -3051,7 +3049,7 @@
       }
     },
     "Clipboard_Access" : {
-      "comment" : "does this message mean that the app is checking hosts availability(like toast message) or a content of button that user can interact?",
+      "comment" : "does this message mean that the app is checking clipboard access or a content of button that user can interact?",
```

Comment:
Yes when the user wants to paste something on WebSSH. iOS asks the user to allow / deny the access.